tests/session-store.test.js

```diff
@@ -0,0 +1,153 @@
+const assert = require('chai').assert;
+const { promisify } = require('util');
+
+const plSessionStore = require('../lib/session-store');
+const sessionStoreExpire = require('../cron/sessionStoreExpire');
+const helperDb = require('./helperDb');
+const sqldb = require('@prairielearn/postgres');
+
+let store = new plSessionStore();
+
+describe('session-store', function () {
+  this.timeout(20000);
+
+  before('set up testing database', helperDb.before);
+  after('tear down testing database', helperDb.after);
+
+  describe('basic functions', () => {
+    it('set() 5 sessions', async () => {
+      await promisify(store.set)('sid1', { value: 1 });
+      await promisify(store.set)('sid2', { value: 2 });
+      await promisify(store.set)('sid3', { value: 3 });
+      await promisify(store.set)('sid4', { value: 4 });
+      await promisify(store.set)('sid5', { value: 5 });
+    });
+
+    it('all() returns list of 5', async () => {
```

Comment:
You should either rewrite these tests to not share state between them or use [`mocha-steps`](https://github.com/rprieto/mocha-steps) to ensure that we abort on the first error. You can find some examples of `mocha-steps` usage in other PL tests.